onadata/libs/tests/serializers/test_geojson_serializer.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.test import APIRequestFactory
+from django.contrib.gis.geos import GeometryCollection, Point
+
+from onadata.libs.serializers.geojson_serializer import GeoJsonSerializer
+from onadata.apps.api.tests.viewsets.test_abstract_viewset import TestAbstractViewSet
+
+
+class TestGeoJsonSerializer(TestAbstractViewSet):
+    def setUp(self):
+        self._login_user_and_profile()
+        self.factory = APIRequestFactory()
+
+    def test_geojson_serializer(self):
+        self._publish_xls_form_to_project()
+        data = {
+            "xform": self.xform.id,
+            "geom": GeometryCollection(Point([6.707548, 6.423264])),
+        }
+        request = self.factory.get("/", **self.extra)
+        serializer = GeoJsonSerializer(data=data, context={"request": request})
+        self.assertTrue(serializer.is_valid())
+        geometry = serializer.validated_data["geom"]
+        self.assertEqual(geometry.geojson, data.get("geom").geojson)
```

Comment:
Should this be testing the simple style spec or both? I think the simple style spec test is missing since the query param `simple_style` is not True by default.

Reply:
@ukanga addressed via https://github.com/onaio/onadata/pull/2255/commits/ed544c4fdd1b5f4a258d4ef72a8f6a51c4013351